src/main/scala/io/mesosphere/mesosaurus/Mesosaurus.scala

```diff
@@ -175,7 +186,8 @@ object Mesosaurus extends Logging {
                 throw new ArgumentParserException("mem standard deviation must be > 0", parser)
             }
             val port = getInt(options, PORT, DEFAULT_PORT)
-            return (master, failover, port, new TaskGenerator(tasks, duration, durationSigma, arrival, load, cpus, cpusSigma, mem, memSigma))
+            return (master, failover, port, new TaskGenerator(tasks, duration,
+                durationSigma, arrival, load, cpus, cpusSigma, mem, memSigma, failRate = fail))
```

Comment:
Why is only one option pass-by-name?  Another hint that the option needs a more descriptive name?